modules/compliance-tests/src/smithy4s/compliancetests/internals/package.scala

```diff
@@ -69,33 +72,68 @@ package object internals {
   }
 
   /*
-       This function takes a string and splits it on a comma delimiter and prunes extra whitespace which
+       This function takes a string that is encoded like source code and splits it on a comma delimiter and prunes extra whitespace which
        what makes it a bit more complicated is we need to keep track of if we are in an open quote or not
    */
-  private[compliancetests] def parseList(s: String): List[String] = {
-    s.foldLeft((Chain.empty[String], 0, 0, false)) {
-      case ((acc, begin, end, quote), elem) =>
-        elem match {
-          // we are in a quote so we negate the quote state and move on
```

Comment:
I'd like to see comments like in the previous version

Reply:
Sure